apps/explorer/src/app/routes/network-parameters/network-parameters.tsx

```diff
@@ -69,7 +70,7 @@ export const NetworkParameterRow = ({
         'group target:bg-vega-pink target:text-white dark:target:bg-vega-yellow dark:target:text-black'
       }
     >
-      {key}
+      {formatLabel(key)}
```

Comment:
formatting network parameters keys. they are in this format `Governance.Proposal.Asset.MaxClose` and they will be shown as `Governance Proposal Asset Max Close`